psse/psse-model/src/main/java/com/powsybl/psse/model/pf/io/BusData.java

```diff
@@ -23,13 +23,21 @@
  * @author José Antonio Marqués {@literal <marquesja at aia.es>}
  */
 class BusData extends AbstractRecordGroup<PsseBus> {
+    private static final String STR_NAME = "name";
+    private static final String STR_BASKV = "baskv";
+    private static final String STR_IDE = "ide";
+    private static final String STR_AREA = "area";
+    private static final String STR_ZONE = "zone";
+    private static final String STR_OWNER = "owner";
+    private static final String STR_VM = "vm";
+    private static final String STR_VA = "va";
```

Comment:
create a new e.g. `PsseIoConstants` utility class with all these constants, we often find the same constants in multiple files.